topostats/grainstats.py

```diff
@@ -441,8 +424,8 @@ def calculate_radius_stats(self, edges: list, points: list) -> tuple[float]:
         return {
             "min": np.min(radii),
```

Comment:
Curious why `np.min()` and `np.max()` don't need explicitly converting to `float()` but `np.mean()` and `np.median()` do. If anything I would have expected it to be the other way round!

Reply:
I believe so? Here there is code in `processing.py` > `get_out_paths` that does it:

<img width="884" alt="image" src="https://github.com/user-attachments/assets/e27cbb1f-d118-4473-b8a3-dbac6b10638c" />


Which is ran as the first step in `processing.py` > `process_scan`:
<img width="779" alt="image" src="https://github.com/user-attachments/assets/4051bb57-6fa0-4546-a1d2-b40efd0dc8bd" />


Might be understanding this wrong though ^^